### PR TITLE
Final batch of fixes

### PR DESCRIPTION
**Pull request by jhrcek**
I finished the whole tutorial. Thanks again for putting this together. Although I like purescript, without your effort I wouldn't have found time and courage to dive into Halogen! I'm looking forward to applying my new knowledge in my next hobby project :smile:

### CODE REVIEW

File: src/06-Driver/14-Subscribing-to-Messages.purs [hunk 1]
```diff
@@ -35,7 +35,7 @@ main =
 
       io.subscribe $ CR.consumer \raisedIntVal -> do
         liftEffect $ log $
-          "Consumer 1: Button was clicked and produced a random integer \
+          "Consumer 2: Button was clicked and produced a random integer \
```
Comment (jhrcek) [inline src/06-Driver/14-Subscribing-to-Messages.purs:38]: Both consumers had Consume 1 which was confusing..

File: src/06-Driver/15-Disposing-Components.md [hunk 2]
```diff
@@ -9,5 +9,4 @@ To compile the next file and view its results in the browser, use these instruct
 ```bash
 spago bundle-app -m Driver.DisposingComponents -t assets/driver/disposing-components.js
 parcel serve assets/driver/disposing-components.html -o disposing-components--parcelified.html --open
-# After the page opens, check your browser's console to see the messages
```
Comment (jhrcek) [inline src/06-Driver/15-Disposing-Components.md:12]: This comment was false - that example has no messages in console.

File: src/06-Driver/14-Subscribing-to-Messages.purs (hunk 1 above)
Comment (JordanMartinez) [reply src/06-Driver/14-Subscribing-to-Messages.purs:38]: Ah, another copy and paste typo. Thanks for fixing that!

File: src/06-Driver/15-Disposing-Components.md (hunk 2 above)
Comment (JordanMartinez) [reply src/06-Driver/15-Disposing-Components.md:12]: Yup! Thanks

**Pull request merged**